core/src/main/resources/error/error-classes.json

```
@@ -134,6 +134,97 @@
         "message" : [
           "Error instantiating GRPC interceptor: <msg>"
         ]
+      },
+      "INVALID_COMMAND_WRITE_INVALID_BUCKET_BY" : {
+        "message" : [
+          "BucketBy must specify a bucket count > 0, received <bucket> instead."
+        ]
+      },
+      "INVALID_PLAN_INPUT" : {
+        "message" : [
+          "Error transforming Spark Connect plan: <msg>"
+        ]
+      },
+      "INVALID_PLAN_UNKNOWN_RELATION": {
```

Wrong the formatting here and bellow:
```suggestion
      "INVALID_PLAN_UNKNOWN_RELATION" : {
```

---

I regenerated the json file based on the SparkThrowableSuite.